doc/user/data/load_save_images.md

```diff
@@ -0,0 +1,446 @@
+# Image data loading
+
+If the STB image library is available on the system (`stb_image.h` and
+`stb_image_write.h` must be available on the compiler's include search path),
+then mlpack will define the `MLPACK_HAS_STB` macro, and support for loading
+individual images or sets of images will be available.
```

Comment:
Actually this is always available now, since we bundle it.  So I think this paragraph is not needed.

Reply:
It's true, we use the system STB if it is available, but if it is not available we use the bundled version.  Therefore support for loading images is *always* available and we don't need this paragraph.

If the user wants to worry about which STB version is going to be used, this is documented in the [compile options](https://www.mlpack.org/doc/user/compile.html#configuring-mlpack-with-compile-time-definitions) section.